### ISSUE TITLE
Blood cell example not working

### ISSUE THREAD

**Issue by franksRsrch** [experiencor/basic-yolo-keras#188]
Hi,

I've been trying to get the blood cell example to work, but it quits out after just a few epochs saying "val_loss did not improve". I have tried increasing the patience to 6, too.

First, I set the warmup epochs to 100 (as in the example) and run from scratch (no loading pre-trained weights). That improves after about 5 epochs, but after that says "val_loss did not improve". It then quits after the loss has not improved for a while.

After that, I load the saved weights from the warmup, and run without any warmup epochs. It improves further for about 3 epochs, after which the loss fails to improve and it quits.

As you'd expect - given the above-  when I then try to use the network, it does not provide any predictions above the threshold.

I'm using the exact code / images / annotation from the example, so can't work out what I'm doing wrong. Any help would be very much appreciated.

**Comment by experiencor** [comment]
Hi @franksRsrch,

1. What is the model that you use? Please use FullYolo for best performance.

2. May you try only 3 warmup epochs? Too many warmup epochs are not good.

**Comment by zeynali** [comment]
sir @experiencor , please more details explain about how running these codes and put you'r system that you have been tested pleaseeeeeeeeeeeeeeee.
 No one has concluded so far expect you.

**Comment by franksRsrch** [comment]
Thanks for the reply @experiencor 

1. I am using the model that is already specified in the blood cell example. The model is hard-coded in that example, rather than specified in the json file. It looks a like a full model.

2. I've tried 3 warmup epochs and get the same result. I'm simply using the code provided in the repository in the examples folder.

**Comment by XdpAreKid** [comment]
The same problem.but i don't know how to fix it.

**Comment by experiencor** [comment]
@franksRsrch @XdpAreKid Please try again. I modified the code to handle warming up automatically.
